npm/vue2/patches/@vue+test-utils+1.3.0.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/node_modules/@vue/test-utils/types/index.d.ts b/node_modules/@vue/test-utils/types/index.d.ts
+index cc15565..9f10c1b 100644
+--- a/node_modules/@vue/test-utils/types/index.d.ts
++++ b/node_modules/@vue/test-utils/types/index.d.ts
+@@ -1,7 +1,9 @@
+ import Vue, { VNodeData, ComponentOptions, FunctionalComponentOptions, Component, RenderContext } from 'vue'
+-import { DefaultProps, PropsDefinition } from 'vue/types/options'
```

Comment:
Since Test Utils is a devDep should we make this a dev patch? https://github.com/ds300/patch-package#dev-only-patches

Reply:
Edit... I published a new Test Utils version (v1.3.1) that has this patch. https://github.com/vuejs/vue-test-utils/releases/tag/1.3.1

What about using that?

Reply:
Thanks @lmiller1990, will update and remove the patch!

Reply:
Done in [01be0bc](https://github.com/cypress-io/cypress/pull/24415/commits/01be0bce155a89bae2845ca0d4154de955a8c199)